### PR TITLE
feat: add icon with menu in tray

### PR DESCRIPTION
**Pull request by ddubrava**
Issue: Custom tray menu #11
Node: v10.15.3

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,3748 @@
+{
```
Comment (DanielHix) [inline package-lock.json]: If you would get rid of package-lock.json as we use Yarn to install packages and not NPM. Make sure you follow the instructions and use Yarn install to install your packages.

File: package.json [hunk 2]
```diff
@@ -40,4 +40,4 @@
     "jquery": "^3.3.1",
     "popper.js": "^1.14.7"
   }
-}
```
Comment (DanielHix) [inline package.json]: Make sure you only commit changes that you actually changed and not things like this. Please remove this change por favor (please).

**Comment by ddubrava** [comment]
Also fix Issue: Minimize to tray when closed #9

File: package-lock.json (hunk 1 above)
Comment (ddubrava) [reply package-lock.json]: ok, fixing

File: package.json (hunk 2 above)
Comment (ddubrava) [reply package.json]: its npm, sry, didnt see

Comment (DanielHix) [reply package.json]: No problem, thanks for contributing.

**Comment by ddubrava** [comment]
Modify commit, return package.json to initial state

**Comment by DanielHix** [comment]
Is this showing a merging is blocked for you?

**Comment by ddubrava** [comment]
i signed the commits. should work now

**Comment by DanielHix** [comment]
Make sure to check [this guide](https://help.github.com/en/articles/generating-a-new-gpg-key) on signing commits, and if you're having trouble with git I'd suggest a GUI depending on if you're comfortable with command line [Github Desktop](https://desktop.github.com/) and [here is](https://github.com/desktop/desktop/issues/78#issuecomment-460251896) a comment on signing commits using it.

**Comment by ddubrava** [comment]
Verifed now. mixed up the emails

**Comment by DanielHix** [comment]
This closes #9 and closes #11

**Pull request merged**